NEWS.md

```diff
@@ -1,5 +1,25 @@
 # News
 
+## dev
+
+### Breaking changes
+
+* Simplified rendering for Quarto websites. Previously, the website was rendered into `_quarto/_site` and manually copied over to `docs/`. The new version removes this logic and instead uses the `output-dir` project option. To transition, change `quarto_website.yml` to:
+  ``` yml
+  project:
+    output-dir: ../docs/
+  ```
```

Comment:
Sorry for being nitpicky (or if I missed something) but IIUC this isn't a breaking change, right? Or if someone has already a Quarto website made with `altdoc`, will it still work with this version?

If it's not please put this in "New features"

Reply:
It is a breaking change. Previously rendered site would be put in _quarto/_site and copied over manually. The update does not copy the files instead relying on quarto to render them in the right location

Reply:
Alright, thanks for the contribution!